packages/experience-builder-types/src/constants.ts

```diff
@@ -12,6 +12,7 @@ export const OUTGOING_EVENTS = {
   NewHoveredElement: 'newHoveredElement',
   ComponentSelected: 'componentSelected',
   RegisteredComponents: 'registeredComponents',
+  DesignTokens: 'registeredDesignTokens',
```

Comment:
This should probably indicate the action so `RegisterDesignTokens` or `DesignTokensRegistered` might be better. (Unfortunately we have not agreed on a specific pattern for the rest of the events so they are inconsistent with each other.)